packages/webapp/vitest.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig, mergeConfig } from 'vitest/config';
+import viteConfig from './vite.config';
+
+export default mergeConfig(
+  viteConfig,
+  defineConfig({
+    test: {
+      include: ['src/tests/**/*.test.js?(x)'],
+      environment: 'happy-dom',
```

Comment:
Added this so that the tests run in a browser environment. In the case of my function since it uses i18n, it fails unless run in a browser environment.